libr/debug/debug.c

```diff
@@ -661,13 +660,47 @@ R_API const char *r_debug_reason_to_string(int type) {
 	case R_DEBUG_REASON_TRAP: return "trap";
 	case R_DEBUG_REASON_SWI: return "software-interrupt";
 	case R_DEBUG_REASON_INT: return "interrupt";
-	case R_DEBUG_REASON_FPU: return "fpu";
```

Comment:
why segfault and fpu are gone?

Reply:
because these can be considered as signal?

Reply:
a division by zero is not a signal. its an actual hardware exception generated by the cpu